bit_manipulation/hamming_distance.cpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file
+ * @brief Returns the Hamming distance between two integers
+ *
+ * @details
+ * To find hamming distance between two integers, we take their xor, which will
+ * have a set bit iff those bits differ in the two numbers.
+ * Hence, we return the number of such set bits.
+ *
+ * @author [Ravishankar Joshi](https://github.com/ravibitsgoa)
+ */
+
+#include <iostream>
+using std::cout, std::endl;
+
+unsigned int bitCount(unsigned int value) {
```

Comment:
Missing function documentation (this is **necessary** and **required**).
See the [typical structure](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) of a program for more information. 🙂